scripts/pre-install.js

```diff
@@ -169,30 +183,45 @@ if (platform === 'win32' && os.arch() !== 'ia32') {
     process.exit(1);
 }
 
-// Check if nrfjprog libraries are working or not
+let isInstallationRequired = false;
 getLibraryVersion()
     .then(version => {
-        console.log('Found nrfjprog libraries at version', version);
+        if (version.major < REQUIRED_VERSION.major ||
```

Comment:
If version is 9.7.1 and 9.7.3 is required then we install. If version is 9.8.1 and 9.7.3 is required, should we install?

Reply:
@bence good catch